app/views/recipe_foods/_form.html.erb

```diff
@@ -0,0 +1,32 @@
+<%= form_with(model: recipe_food, class: "contents") do |form| %>
+  <% if recipe_food.errors.any? %>
+    <div id="error_explanation" class="bg-red-50 text-red-500 px-3 py-2 font-medium rounded-lg mt-3">
+      <h2><%= pluralize(recipe_food.errors.count, "error") %> prohibited this recipe_food from being saved:</h2>
+
+      <ul>
+        <% recipe_food.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="my-5">
+    <%= form.label :quantity %>
+    <%= form.number_field :quantity, class: "block shadow rounded-md border border-gray-200 outline-none px-3 py-2 mt-2 w-full" %>
+  </div>
+
+<div class="my-5">
+    <%= form.label :Food_id, "Enter Food Id " %>
+    <%= form.number_field :food_id, class: "block shadow rounded-md border border-gray-200 outline-none px-3 py-2 mt-2 w-full" %>
+  </div>
+
+  <div class="my-5">
+    <%= form.label :recipe_id, "Enter Recipe Id"%>
```

Comment:
- [ ] `Enter Recipe Id` should be autoadded in controller, users should not be filling this field manually. The id of recipe should be taken from params, otherwise users can enter different id and the `New recipe food` will belong to different recipe 😉